src/main/ruby/core/rbconfig.rb

```diff
@@ -190,6 +190,10 @@ def RbConfig.expand(val, config = CONFIG)
     val
   end
 
+  if ruby_home
+    expanded['LDSHARED'] = "#{RbConfig.ruby} -Xgraal.warn_unless=false #{libdir}/cext/ldshared.rb"
```

Comment:
This is not very nice, but I'll clean it up when moving RbConfig out of core in another PR.
It's needed by RubyInline (`rbconfig-for-mkmf` is not automatically required).